recipes/eigen/bld.bat

```diff
@@ -0,0 +1,6 @@
+mkdir -p build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . --target INSTALL --config Release
```

Comment:
So, I have copied your script @patricksnape and pushed that instead. Maybe @msarahan can tell us what black magic is required here.